proto/executor/v1/executor.proto

```diff
@@ -45,6 +45,13 @@ message ProcessBatchResponse {
     uint64 cumulative_gas_used = 12;
     repeated ProcessTransactionResponse responses = 13;
     Error error = 14;
+    repeated TouchedAddresses touched_addresses = 15;
+}
+
+message TouchedAddresses {
+    string address = 1;
+    string nonce = 2;
+    string balance = 3;
```

Comment:
If only one of the two fields is updated, how can we differentiate it? 0 values is valid, so how can we know if updated to 0 value or not updated?

Reply:
Comparing with what we already have for that address.

Reply:
I'm not sure what is the exact function of this `touchedAddresses` but an address could also have `hash_bytecode` and `storage` changed

Reply:
We can make them `optional` in the proto, which makes them as pointers in the GoLang and I guess in the other languages must be like that. Which will give us the opportunity to represent the fields which hasn't changed with `nil`/`null`.